plugins/modules/grafana_datasource.py

```diff
@@ -587,13 +632,15 @@ def main():
                               'postgres',
                               'cloudwatch',
                               'alexanderzobnin-zabbix-datasource',
+                              'camptocamp-prometheus-alertmanager-datasource',
                               'sni-thruk-datasource',
                               'loki'], required=True),
         ds_url=dict(required=True, type='str'),
         access=dict(default='proxy', choices=['proxy', 'direct']),
         database=dict(type='str', default=""),
         user=dict(default='', type='str'),
         password=dict(default='', no_log=True, type='str'),
+        secure_password=dict(default='', no_log=True, type='str'),
```

Comment:
Yes it could. In this case I just saw that a dedicated property might be a good idea in conjunction to `password:` to make it clear the later one is unencrypted while the new one is.